### PR TITLE
Add Entry: HTML Tags: <blink>

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Add entry for the old HTML blink tag prompted by the Target Keywords list.

Assumes the Docs tag 'Obsolete' is added to the tag list from PR  606

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.

### CODE REVIEW

File: content/html/concepts/tags/terms/blink/blink.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - 'Style'
+  - 'Obsolete'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<blink>` tag is an obsolete tag that was used historically to make the text it contained flash on and off. It is non-standard, depreciated, and not part of the HTML specifications and should not be used.
+
+Using this element is bad design practice and is discouraged by accesability standards. The CSS specification allows browsers to ignore this tag.
```
Comment (yangc95) [inline content/html/concepts/tags/terms/blink/blink.md:18]:
```suggestion
Using this element is a bad design practice and is discouraged by accessibility standards. The CSS specification allows browsers to ignore this tag.
```

File: content/html/concepts/tags/terms/blink/blink.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Tags:
+  - 'Tags'
+  - 'Style'
+  - 'Obsolete'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<blink>` tag is an obsolete tag that was used historically to make the text it contained flash on and off. It is non-standard, depreciated, and not part of the HTML specifications and should not be used.
```
Comment (yangc95) [inline content/html/concepts/tags/terms/blink/blink.md:16]:
```suggestion
The `<blink>` tag is an obsolete tag that was used historically to make text flash on and off. It is non-standard, depreciated, and not part of the HTML specifications.
```

File: content/html/concepts/tags/terms/blink/blink.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+Title: '<blink>'
+Description: 'Obsolete tag that originally caused text to flash on and off. It is now depreciated and should not be used.'
```
Comment (yangc95) [inline content/html/concepts/tags/terms/blink/blink.md:3]:
```suggestion
Description: 'Obsolete tag that originally caused text to flash on and off. It is now deprecated and non-standard.'
```

File: content/html/concepts/tags/terms/blink/blink.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+The `<blink>` tag is an obsolete tag that was used historically to make the text it contained flash on and off. It is non-standard, depreciated, and not part of the HTML specifications and should not be used.
+
+Using this element is bad design practice and is discouraged by accesability standards. The CSS specification allows browsers to ignore this tag.
+
+## Syntax
+
+```html
+<!-- Don't use this tag! -->
+<blink><!-- blinking text goes here --></blink>
```
Comment (yangc95) [inline content/html/concepts/tags/terms/blink/blink.md:24]:
```suggestion
<blink><!-- Blinking text goes here --></blink>
```

File: content/html/concepts/tags/terms/blink/blink.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+The `<blink>` tag is an obsolete tag that was used historically to make the text it contained flash on and off. It is non-standard, depreciated, and not part of the HTML specifications and should not be used.
+
+Using this element is bad design practice and is discouraged by accesability standards. The CSS specification allows browsers to ignore this tag.
+
+## Syntax
+
+```html
+<!-- Don't use this tag! -->
```
Comment (yangc95) [inline content/html/concepts/tags/terms/blink/blink.md:23]:
```suggestion
<!-- Not advised to use this tag! -->
```

**Review by yangc95 — COMMENTED** [review]
Hi @SSwiniarski ! Review 1️⃣ in for this. Looks like a quick merge entry. No particular technical suggestions, just a few word choice suggestions, and minor things.

**Comment by SSwiniarski** [comment]
@yangc95, I implemented your suggestions.

**Comment by Dusch4593** [comment]
Cool cool! I'll go ahead with review 2️⃣

File: content/html/concepts/tags/terms/blink/blink.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  - 'Style'
+  - 'Obsolete'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<blink>` tag is an obsolete tag that was used historically to make text flash on and off. It is non-standard, depreciated, and not part of the HTML specifications.
+
+Using this element is a bad design practice and is discouraged by accessibility standards. The CSS specification allows browsers to ignore this tag.
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/blink/blink.md:18]:
```suggestion
The `<blink>` tag was used historically used to make text flash on and off. However, it is now considered to be obsolete and non-standard. It is now, or in the middle of being, depreciated from most browsers is not a part of the HTML specifications.

Using this tag is a bad design practice and is heavily discouraged by accessibility standards. The CSS specification allows browsers to ignore this tag.
```

I feel like there are some things that we could/should link to in these paragraphs:

- "HTML specifications" (are we talking about [this](https://html.spec.whatwg.org/)?)
- "accessibility standards" (are we talking about [this](https://www.w3.org/WAI/standards-guidelines/wcag/)?)
- "CSS specification" (most recent snapshot from 2021 appears be [this](https://www.w3.org/TR/css-2021/))

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Finished with my review. Not much to suggest but the last one is more like an opening for a discussion rather than a pure suggested change.

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented yor suggestions, including the links.

File: content/html/concepts/tags/terms/blink/blink.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+---
+Title: '<blink>'
+Description: 'Used to make text to flash on and off and is now obsolete, deprecated, and non-standard.'
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/blink/blink.md:3]:
```suggestion
Description: 'Used to make text flash on and off and is now obsolete, deprecated, and non-standard.'
```

**Review by yangc95 — APPROVED** [review]

**Review by Dusch4593 — APPROVED** [review]
LGTM!

**Comment by Dusch4593** [comment]
@KTom101 Did you want to take a quick look at this before we merge?

**Review by KTom101 — APPROVED** [review]
LGTM

**Comment by Dusch4593** [comment]
Awesome! 🚀 

Once the entry goes live, we can view it here: https://www.codecademy.com/resources/docs/html/tags/blink

**Pull request merged**